common/beerocks/btl/btl_local_bus.cpp

```diff
@@ -82,16 +71,26 @@ bool transport_socket_thread::bus_init()
 
 bool transport_socket_thread::bus_subscribe(const std::vector<ieee1905_1::eMessageType> &msg_types)
 {
-    bool ret = true;
-
     LOG_IF(!bus, FATAL) << "Bus is not allocated!";
     LOG_IF(!poller, FATAL) << "Poller is not allocaed!";
     bus->Init();
```

Comment:
I think we should have a `bus_Init()` which will call `bus->init()` so in the future if we call `bus_subscribe` more than once it will not cause an issue.

Reply:
we have it already.